skins/monobook/main.css

```diff
@@ -1238,3 +1244,43 @@ div.mw-lag-warn-high {
 
 
 /* end Wikia monobook modification */
+
```

Comment:
Please use a SASS mixin for this element to avoid C&P in two files, unless not possible